201-aci-sftp-files/azuredeploy.json

```diff
@@ -0,0 +1,163 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "storageAccountPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Storage account naming prefix"
+            }
+        },
+        "storageAccountType": {
+            "type": "string",
+            "defaultValue": "Standard_LRS",
+            "allowedValues": [
+                "Standard_LRS",
+                "Standard_GRS"
+            ],
+            "metadata": {
+                "description": "Storage account type"
+            }
+        },
+        "fileShareName": {
+            "type": "string",
+            "defaultValue": "sftpfileshare",
+            "metadata": {
+                "description": "Name of file share to be created"
+            }
+        },
+        "sftpUser": {
+            "type": "string",
+            "defaultValue": "sftp",
+            "metadata": {
+                "description": "Username to use for SFTP access"
+            }
+        },
+        "sftpPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Password to use for SFTP access"
+            }
+        }
+    },
+    "variables": {
+        "cliContainerName": "create-share",
+        "cliContainerGroupName": "create-share-group",
+        "cliContainerImage": "microsoft/azure-cli:latest",
+        "sftpContainerName": "sftp",
+        "sftpContainerGroupName": "sftp-group",
+        "sftpContainerImage": "atmoz/sftp:latest",
+        "sftpEnvVariable": "[concat(parameters('sftpUser'), ':', parameters('sftpPassword'), ':1001')]",
+        "storageAccountName": "[concat(parameters('storageAccountPrefix'), uniqueString(resourceGroup().id))]",
+        "createFileShareTemplateUri": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-aci-sftp-files/nestedtemplates/createFileShare.json"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts

Reply:
Removed nesting, so no longer required